pkg/operator/deploy/deploy.go

```diff
@@ -125,18 +123,24 @@ func templateManifests(data deploymentData) ([][]byte, error) {
 
 func (o *operator) createDeploymentData() deploymentData {
 	image := o.env.AROOperatorImage()
-	hasVersion := false
+
+	// HACK: Override for ARO_IMAGE env variable setup in local-dev mode
+	version := "latest"
```

Comment:
This will override any previous "unknown" value in local dev to "latest".  

So if you're running local dev mode, you're image tag will always be the tag of the image, or latest if there is no tag.  This should allow us to run the admin update tasks Srinivas introduced and still be successful.